tests/test_clip_intensity_percentiles.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.transforms import ClipIntensityPercentiles
+from monai.transforms.utils import soft_clip
+from monai.transforms.utils_pytorch_numpy_unification import clip
+from tests.utils import TEST_NDARRAYS, NumpyImageTestCase2D, assert_allclose
+
+
+class TestClipIntensityPercentiles(NumpyImageTestCase2D):
```

Comment:
Please add at least one test example with 3D arrays. Since a lot of medical images have dimensions higher than two. Thanks.

Reply:
Hello, of course, I added 3D tests in the new commit

Reply:
Thanks.